src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+// action types
+const LOAD_BOOKS = 'load';
+const ADD_BOOK = 'add-book';
+const REMOVE_BOOK = 'remove-book';
```

Comment:
- [ ] Kindly consider the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). For example, `const ADD_BOOK = 'bookstore/books/ADD_BOOK'`